src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
It's better to generate calendar days using a loop in a preprocessor or in JavaScript, to avoid repetitive code and to make it easier to maintain and modify the number of days.